src/main/java/com/endava/internship/collections/Student.java

```diff
@@ -1,6 +1,7 @@
 package com.endava.internship.collections;
 
 import java.time.LocalDate;
+import java.util.Objects;
 
 /**
  * The class that defines the element that will be contained by your collection
```

Comment:
Please make variables that you think is unique as final, so no one can change them after creating.
If the person will change his name we will consider him as a different student(and btw you don't have setters so no problem here)

Reply:
Fixed